web-components/src/components/modal/FormModalTemplate.tsx

```diff
@@ -11,7 +11,7 @@ const useStyles = makeStyles(theme => ({
       padding: `${theme.spacing(10.75)} ${theme.spacing(7.5)} ${theme.spacing(
         12.5,
       )}`,
-      maxWidth: theme.spacing(140),
+      maxWidth: theme.spacing(148),
```

Comment:
Take into account scrollbar width to match Erika's designs.

Reply:
nit; should we set a constant here? i.e. `SCROLL_BAR_WIDTH`, and then say `140 + SCROLL_BAR_WIDTH`